model/trace/gen-ai.yaml

```diff
@@ -85,3 +83,33 @@ groups:
           conditionally_required: if and only if corresponding event is enabled
         note: >
           It's RECOMMENDED to format completions as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
+
+  - id: trace.gen_ai.client
+    extends: trace.gen_ai.client.common
+    brief: >
+      Describes a GenAI operation span.
+    attributes:
+      - ref: gen_ai.request.top_k
+        requirement_level: recommended
+
+  - id: trace.gen_ai.openai.client
+    extends: trace.gen_ai.client.common
+    brief: >
+      Describes an OpenAI operation span.
+    attributes:
+      - ref: gen_ai.openai.request.seed
+        requirement_level:
+          conditionally_required: if the request includes a seed
+      - ref: gen_ai.openai.request.response_format
+        requirement_level:
+          conditionally_required: if the request includes a response_format
+      - ref: gen_ai.openai.request.service_tier
+        requirement_level:
+          conditionally_required: if the request includes a service_tier
```

Comment:
maybe we can skip it if it has a default value?  - it'll be set on the response anyway.

```suggestion
          conditionally_required: if the request includes a service_tier and the value is not `auto`
```

> [When not set, the default behavior is 'auto'.](https://platform.openai.com/docs/api-reference/chat/create)

(ironically they have a `default` value which is not a default value 😄 )